src/plugins/discourse/mirrorRepository.test.js

```diff
@@ -24,4 +25,61 @@ describe("plugins/discourse/mirrorRepository", () => {
     expect(() => new SqliteMirrorRepository(db, url1)).not.toThrow();
     expect(fs.readFileSync(filename).toJSON()).toEqual(data);
   });
+
+  it("bumpedMsForTopic finds an existing topic's bumpedMs", () => {
+    // Given
+    const db = new Database(":memory:");
+    const url = "http://example.com";
+    const repository = new SqliteMirrorRepository(db, url);
+    const topic: Topic = {
+      id: 123,
+      categoryId: 1,
+      title: "Sample topic",
+      timestampMs: 456789,
+      bumpedMs: 456789,
```

Comment:
given that bumpedMs and timestampMs are the same value, this test can't distinguish whether it's returning the bumpedMs or the timestampMs.

Reply:
I defaulted to it, as it's what a real world topic would look like with just 1 post. But fair point, so I've changed to having a bump in mind for this and the other tests with a similar issue.